data/brands/shop/supermarket.json

```diff
@@ -30,6 +30,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "Reliance Smart Super Store",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Mall","Jio Mall","Jio Supermarket"],
+      "tags": {
+        "brand": "Reliance Smart Super Store",
```

Comment:
Is the brand "Reliance Smart" or "Reliance Smart Super Store"? https://relianceretail.com/reliance-smart.html would imply that it's just "Reliance Smart" (ignoring their choice of capitalization).